app/views/paths/index.html.erb

```diff
@@ -1,65 +1,75 @@
 <%= title("All Paths") %>
 
-<div class="paths curriculum-background">
-  <div class="container tmp-container-spacer">
-    <h1 class="page-heading-title">All Paths</h1>
-    <% @paths.each do |path| %>
-      <% if path.title === 'Foundations' %>
-        <div class='card-main card-main--clickable'>
-          <%= link_to '', path, class: 'card-main__link' %>
-          <div class="card-main__header">
-            <div>
-              <h2 class="foundations-title mb-2"><%= path.title %></h2>
-              <h3 class="text-2xl">Start here!</h3>
-            </div>
-            <% if current_user.nil? %>
-              <%= link_to 'View Path', path_url(path), class: 'button button--secondary px-10 card-main__button' %>
-            <% elsif current_user.path === path %>
-              <%= link_to 'Resume', path_url(path), class: 'button button--primary px-10 card-main__button', data: { test_id: "#{path.title.downcase}-resume-path-btn"  } %>
-            <% else %>
-              <%= button_to 'Select', users_paths_path(path_id: path.id), class: 'button button--secondary px-10 card-main__button', remote: false, method: :post, data: { disable_with: 'Selecting...' } %>
-            <% end %>
-          </div>
-          <div class="card-main__description">
-            <% if path.prerequisites.any? %>
-              <p class="card-main__prerequisites">
-                <strong>Prerequisites: <%= prerequisites_list_for(path).html_safe %></strong>
-              </p>
-            <% end %>
-            <p><%= path.description%></p>
-          </div>
+<div class="content-container-lg">
+  <h1 class="page-heading-title">All Paths</h1>
+
+  <%= render CardComponent.new(classes: 'odin-dark-bg-accent') do |card| %>
+    <% card.header(classes: 'flex justify-between items-center flex-col sm:flex-row space-y-4 sm:space-y-0') do %>
+      <div class="flex flex-col space-y-5 sm:space-x-6 sm:space-y-0 sm:flex-row items-center">
+        <%= link_to path_path(@default_path) do %>
```

Comment:
The `path_path` always makes me cringe haha. We didn't take the route helper naming into account when using the path naming.

I decided to go with `foundations` for the name since that's what the default path is. `@path` on its own wasn't descriptive enough imo.

Reply:
Although I was half joking with the `@default_path` name, I actually really like the change to `@foundations`, the domain seems much clearer and the views read much better (in my mind, at least) :ok_hand: